src/components/views/avatars/MemberAvatar.tsx

```diff
@@ -77,8 +76,8 @@ export default function MemberAvatar({
 
         if (!title) {
             title =
-                UserIdentifierCustomisations.getDisplayUserIdentifier(member?.userId ?? "", {
-                    roomId: member?.roomId ?? "",
+                UserIdentifierCustomisations.getDisplayUserIdentifier!(member.userId, {
```

Comment:
This is interesting :-) Why the `!` here?

Reply:
This wasn't really central to this change-set, so I just left the behavior what it was. If `getDisplayUserIdentifier` is `undefined`, it would have errored, as it does now.

There's an argument to be made for optional chaining being "more correct". In reality, [`getDisplayUserIdentifier` is *always* defined](https://github.com/matrix-org/matrix-react-sdk/blob/a69b805156126708c78e2a0ed7c5fab3919a7f82/src/customisations/UserIdentifier.ts#L17-L41), but I don't really know enough about the customisation system to be able to say how universal that is.